dev/com.ibm.ws.microprofile.metrics.2.0/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops

```diff
@@ -128,6 +128,11 @@ session.activeSessions.description=The number of concurrently active sessions. (
 session.invalidated.total.description=The number of sessions that have logged out since this metric was enabled.
 session.invalidatedbyTimeout.total.description=The number of sessions that have logged out by timeout since this metric was enabled.
 
+requestTiming.requestCount.description=The number of servlet requests since the server started.
+requestTiming.activeRequestCount.description=The number of servlet requests currently running.
+requestTiming.slowRequestCount.description=The number of servlet requests that are currently running and are deemed slow.
```

Comment:
Suggested change: Replace "and are deemed slow" with "but are slow" because a server/machine/process cannot "deem." (I'm assuming that a non-human entity is deciding whether to show the message.)

Reply:
fixed